internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/event.go

```diff
@@ -174,15 +174,17 @@ func SendEvent(
 			"streamId":  params.SourceKey.String(),
 			"sinkId":    params.SinkID,
 		},
+		Results: map[string]any{},
 	}
 	if params.SourceName != "" {
 		event.Params["sourceName"] = params.SourceName
 	}
 
 	var sErr error
 	defer func() {
-		// BC compatibility, should be removed later.
-		event.Results = event.Params
+		if len(event.Results) == 0 {
+			event.Results = nil
+		}
```

Comment:
Why did we add this?

Reply:
I initialize the `Results: map[string]any{},` above but when it's empty it can be removed from the event. It's not technically necessary but the results is marked in the schema of our events as optional.